index.html

```diff
@@ -2766,134 +2728,114 @@ <h3>Consumption</h3>
 suggestions in section 3.9 of the CBOR [[RFC7049]] specification for
 deterministic mappings the following constraints of the DID Document model MUST
 be followed:
-        </p>
+      </p>
         <ul>
-          <li>
-The keys in every map must be sorted lowest value to highest. Sorting is
-performed on the bytes of the representation of the keys.
-          </li>
-          <li>
-Indefinite-length items must be made into definite-length items.
-          </li>
+  <li>Duplicate key in the same CBOR map MUST through and Error.</li>
         </ul>
       </section>
 
+      
+
       <section>
         <h3>CBOR Extensibility</h3>
         <p>
 In CBOR, one point of extensibility is with the use of CBOR tags. [[RFC7049]]
 defines a basic set of data types, as well as a tagging mechanism that enables
 extending the set of data types supported via the <a
-href="https://www.iana.org/assignments/cbor-tags/cbor-tags.xhtml">CBOR Tag
+href="https://www.iana.org/assignments/cbor-tags/cbor-tags.xhtml">IANA CBOR Tag
 Registry</a>. This allows for tags to enhance the semantic description of the
-data that follows.
-        </p>
+data that follows.  Extensibility with CBOR tags also facilitates lossless conversion to
+othere core representations. CBOR Tags number 21 to 23 indicate that a following byte string 
+might require a specific encoding when interoperating with a text-based
+representation such as JSON.  These tags are useful when an encoder knows that the
+byte string data it is writing is likely to be later converted to a
+particular JSON-based usage.  These three tag numbers suggest conversions to three of the base data
+encodings defined in [[RFC4648]].  Tag number 21 suggests conversion to
+base64url encoding (Section 5 of [[RFC4648]]), where padding is not used
+(see Section 3.2 of [[RFC4648]]); that is, all trailing equals signs
+("=") are removed from the encoded string.  Tag number 22 suggests
+conversion to classical base64 encoding (Section 4 of [[RFC4648]] ), with
+padding as defined in [[RFC4648]].  For both base64url and base64,
+padding bits are set to zero (see Section 3.5 of [[RFC4648]] ), and
+encoding is performed without the inclusion of any line breaks,
+whitespace, or other additional characters.  Tag number 23 suggests
+conversion to base16 (hex) encoding, with uppercase alphabetics (see
+Section 8 of [[RFC4648]]).  Note that, for all three tag numbers, the
+encoding of the empty byte string is the empty text string of other representations.
+</p>
 
```

Comment:
I don't understand what I would need to do as an implementer with this text. How does my implementation change? I don't think any of this PR suggests that we use tags 21-23, so why are we talking about them here?

Reply:
CBOR tags facilitate a `lossless` conversion between core representations with use of semantic sugar.  It would be helpful to implementors who need to represent binary data and give `batteries included` clear instruction for how to `losslessly` convert to other representations during DID Resolution.   This section highlights the method to convert between core representations.

Reply:
I understand the desire to provide a means for possible lossless conversion to another representation, but am unsure what this description has to do with CBOR Extensibility. Perhaps the section should be re-named?